what_is.md

```diff
@@ -120,10 +120,12 @@ for mut in mutated_ts.mutations():  # This entire loop is just to make pretty la
     prev = mutated_ts.mutation(mut.parent).derived_state if older_mut else site.ancestral_state
     mut_labels[mut.id] = l.format(site.position, prev, mut.derived_state)
 
-SVG(mutated_ts.draw_svg(size=sz, node_labels=labels, mutation_labels=mut_labels, style=style1))
+SVG(mutated_ts.draw_svg(
+    size=sz, style=style1,
+    node_labels=labels, mutation_labels=mut_labels)) #, x_label="Chromosome base pair position"))
 ```
 
-The trees tell us that, for example, the final mutation (at position 980) is inherited
+<!-- The mutations on the branches of the trees are also **** -->.The trees tell us that, for example, the final mutation (at position 980) is inherited
```

Comment:
What's the story here? If you want to put in comments, [this](https://myst-parser.readthedocs.io/en/latest/using/syntax.html#syntax-comments) is the way to do it.

Reply:
Hmm, interesting that the html comment stuff works (I used it later on). I've used it as a way to block comment out a whole section of text. I can't see a simple way to do that to a block, or inline in a sentence, using `#`? But you're right that I also can't see any info in the JupyterBook docs about `<!-- -->`.

Reply:
Ah, I see. `<!--` is [acceptable markdown](https://spec.commonmark.org/0.29/#html-blocks) within MyST.